### DISCUSSION TITLE
Touchscreen recommendations

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#271] [Ideas]
Questions about which touchscreen to use are still quite common.

Here is one good screen many seems to be overlooking (even though it is in the documentation):

https://bareboat-necessities.github.io/my-bareboat/bareboat-os.html#_touchthink_marine_hdmi_touchscreen

Fair Winds!

**Comment by EarlWer** [comment]
If I could offer a suggestion as a newbie, looking for a compatible touchscreen for my RPi 4B project...

What about communicating with the supplier to write up the exact specs of the touchscreen and keep the details current?
The BBN page that mentions the product also says "but you have to spec out your order precisely". The other problem is that the info is not dated, so could be years old...

The vendor search at https://www.touchtecs.com cannot find 'TPC-CD-M101'.
The vendor site has hundreds of products and the specs you listed are great, but it's going to take hours/days to find and communicate my order.

So, if you could select a few configurations (screen size, color, whatever) and post exact ordering details, it would make finding compatible touchscreens much easier.

I'm sure the supplier would be happy to create a 'BBN-TP101-HDMI' (or whatever) product code, so that buyers from BBN would have no problems getting a compatible touchscreen with no hassles...  and maybe a commission for you.

It would also solve the issue of common touchscreen questions. A sticky post with exact product numbers would be really useful.

thanks.  Looking forward to my BBN installation ;-)

**Reply by mgrouch** [reply]
Contact 

sales@sztouchtec.com

attention to YoYo Lee. Mention Bareboat Necessities and they should offer you quote for the correct 10’ screen. 

There was number of people referring to it including me. 

If you plant to flush mount it on a panel you might ask for a bezel for it too.

**Reply by mgrouch** [reply]
The screen looks like this:

https://m.alibaba.com/product/1600335180253/Touchthink-10-1-marine-application-outdoor.html

**Reply by EarlWer** [reply]
Hi Mikhael,
YoYo has left the company and has been replaced by Pauer. They sent me the specs (below), but it has VGA output and only 300 nits.
Is there a simple/reliable way to convert the VGA to HDMI for the Raspberry Pi?
I will ask them if they have a similar model with HDMI and brighter screen.


Specs

Screen Size | 10.1"
Display Resolution & Aspect Ratio | 1080*800(16:10)
Touch Type | Capacitive (Resistive Touch , Non-touch optional)
Contrast Ratio | 600:1
Brightness (CD/㎡) | 300 nits (Can customized up to 1000 nits)
Viewing Angles (CR =10) | 85/85/85/85 (L/R/U/D)
Response Time(Typ.Tr+Tf) | 6.5ms
Main I/O Interfaces | 1*DC  (DC 12-36V)
1*VGA, waterproof circular connector(GX12-15 pin) VGA port
1* Touch, waterproof circular connector(GX12-4 pin) touch input
Operating Temperature | -10~60℃ (Can customized -20~70℃)
Storage Temperature | -20~70℃

**Reply by mgrouch** [reply]
Here is what you need to ask for:

spec out your order precisely:

10.1 inch, 1280*800 16:10

10 points capacitive touch screen monitor, optically bonded

1000nit high brightness, -20 to 70 degree wide use temperature.

industrial front IP67 waterproof dust-proof vehicle use aluminum alloy bezel style.

7H hardness tempered glass grade A

with 12-36V DC.

waterproof aviation ports: HDMI+touch+DC

build-in HDMI speakers (on sides under bezel)

front light sensor, photo sensitive head. change brightness with ambient light

VESA mountable

5 waterproof control buttons on front bezel

front status light indicator

with Aviation port to standard port cables: HDMI cable, with touch cable, DC pigtail power cable, with AC/DC power adapter. The EXW price was ~us$222/unit (without flush mount kit) for 1-50 units.

Ask for a gasket with a panel flush mount kit too.

EU or US (based on your location) AC/DC power adapter.

**Reply by EarlWer** [reply]
Thanks. I'll let you know what happens next ;-)

I forgot to mention how impressed I was with the installation of BBN. I've installed OpenCPN previously and it's still lots of work to get something useful.  I installed BBN and was immediately greeted by a nice screen with 15 icons...    Even the AvNav app, which I will have to try...   I'm looking forward to installing it on my boat.  Great job.
